Feature/add efk stack for monitoring

nipierre opened the pull request:
Added an EFK stack for monitoring.
- added a directory `ek` for the docker-compose of Elasticsearch, Kibana and Grafana
- added Fluentd in the docker-compose of workers
- added an environment variable in `.env` (ELK) to enable/disable (true/false) the stack

Miscellaneous.
- added the automation of the network: host config in order to ease the use of 'external' ex_backend.
- put `.env` and `.env.workers` in `.gitignore` and created `.env.dist` and `.env.workers.dist`.
- removed typos.

mathiaHT commented on scripts/generate_workers_cfg.sh:
```diff
@@ -101,20 +117,50 @@ generate_workers () {
-            add_item mediacloudai_global
-            add_item workers
+
+            if [ $2 = "master" ]; then
```
Did you rename 'master' config by 'startup'?

mathiaHT replied:
Because in the README file, I think you are mentionning only dev and startup config name

nipierre replied:
True ! Gonna correct it !

mathiaHT commented on scripts/generate_workers_cfg.sh:
```diff
@@ -101,20 +117,50 @@ generate_workers () {
+
+            if [ $EFK_FLAG = true ]; then
+              add_section logging
+              depth 1
```
Do you think it's clearer to write `depth 1; add_section_with_value driver "fluentd"` ?

nipierre replied:
Definitely, I'll do the mod, it'll be clearer to everybody

MarcAntoine-Arnaud commented on ek/README.md:
```diff
@@ -0,0 +1,28 @@
+# The EFK stack
```
Why the folder is `ek` and everywhere EFK ?
Do you think we can name the folder `monitoring` ?

nipierre replied:
ek was because fluentd is not created in this directory but in workers. 
I agree that we can call it `monitoring`

MarcAntoine-Arnaud commented on README.md:
```diff
@@ -38,15 +38,26 @@ make workers-generate-cfg
 
+### EFK stack for monitoring
+
+In order to perform an applicative monitoring (viz. save and access the logs of the docker containers), an EFK (Elasticsearch-Fluentd-Kibana) stack is used. More informations on it [here](ek/README.md).
```
Can we replace:
More informations on it here

by something like:
A complete description on it is placed in [monitoring](monitoring/README.md) folder

nipierre replied:
ok

nipierre commented:
_Commit message :_

Added an EFK stack for monitoring.

- added a directory `monitoring` for the docker-compose of Elasticsearch, Kibana and Grafana
- added Fluentd in the docker-compose of workers
-  added an environment variable in `.env` (ELK) to enable/disable (true/false) the stack

Miscellaneous.

- added the automation of the network: host config in order to ease the use of 'external' ex_backend.
- put `.env` and `.env.workers` in `.gitignore` and created `.env.dist` and `.env.workers.dist`.
-  removed typos.
